docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,118 @@ relationships directionally between a cause/exposure and effect/outcome.
   Discuss criteria (sim science and GBD) for modeling a risk outcome pair
 
 Confounding
-""""""""""""
+"""""""""""
+**What is confounding?**
+
+Much of epidemiology is concerned with establishing associations between exposures and health outcomes. This usually involves comparing representative groups from two populations that are as similar as possible in all respects except the exposure(s) under study. **Confounding occurs when an estimate of the association between an exposure and a health outcome is mixed up with the effect of another exposure on the same health outcome, and the two exposures are associated**. When present, it results in a biased estimate of the effect of exposure on the health outcome. The bias can be negative (resulting in underestimation of the exposure effect), or positive (overestimation of the exposure effect), or can even reverse the apparent direction of effect. Confounding should always be addressed when assessing causality and failure to account for confounding leads to incorrectly concluding that an effect or its magnitude is due to one, rather than another variable. 
+
+In order for a factor to be a confounder (C), it must meet the following criteria:
+
+  1. Must be a risk factor, or the surrogate of a risk factor, for the outcome (O) of interest in those who are unexposed to the exposure (E) being investigated.
+  2. Must be correlated, positively or negatively, with the exposure (E). If the study population is stratified into exposed and unexposed groups, this means that (C) has a differential distribution in the two groups.
+  3. In addition, (C) should not be an intermediate step on the causal pathway between the exposure (E) and outcome (O) being investigated (else the factor is considered an :ref:`intermediate <Intermediates>`). In other words, (C) should not be caused by the exposure (E).  
```

Comment:
I think we should also mention mediation here. Maybe something like...

3. In addition, (C) should not be an intermediate on the a causal pathway between the exposure (E) and outcome (O). If this is the case, (C) may in fact be an intermediate [LINK] or a mediator [LINK]. In other words, (C) should not be caused by the exposure (E).

Reply:
The coffee example actually made me a bit confused about this definition---is there any definitional requirement that the exposure be causally related (or plausibly causally related) to the outcome?

Because if not, you could take any variable (C) which is causally related to some variable (O) and associated with some (E), where E and O are otherwise unrelated, and say that C confounds the relationship between E and O. (Which may well be the case, but I wasn't previously thinking of this as something that would fit under the definition of confounding)

Reply:
Yeah, good point. Does the first requirement, that (C) "Must be a risk factor, or the surrogate of a risk factor, for the outcome (O)" imply that (C) must be causally related to (O)?  Because our definition of "risk factor" is "any attribute whose measure is causally related to the measure of an outcome."

Or if (C) is just a "surrogate" of the risk factor, does this mean it's _not_ necessarily causally related to (O)?  What exactly does "surrogate" mean?

Reply:
In the coffee example, coffee is the EXPOSURE, and smoking is the CONFOUNDER, cancer is the OUTCOME. I don't think there is any requirement that the is exposure is causally related to the outcome- as this is the big unknown and what we are trying to prove or disprove! In our example, after controlling for smoking we see NO effect on cancer from coffee so we can say that there is no causal relationship between our exposure and outcome. 

I think we need to think of it as there is an exposure and there is an outcome. People are saying E causes O for some reason (like, vaccines causes autism). We as epidemiologists need to prove yes or no, eg. whether the relationship between vaccines and autism is real (causal) or spurious (acting through confounders only).

Reply:
@NathanielBlairStahn that is correct- the confounder must be causally related (risk factor) for the outcome for it to be considered. 

I think we need to think of it as first we have an exposure outcome pair that we are unsure of (like this new claim that coffee causes cancer, or vaccines causes autism). Then some epidemiologists are like...that's curious, we are unconvinced!! so they postulate, that the relationship that is being observed between coffee and cancer OR vaccines and autism must be _caused_ by "something else" and this "something else" is just _mixed-up_ with the exposure.  To figure out what this "something else" is, they must then come up with a list of things that they know must cause the outcome. This "something else" must also be correlated with the exposure, otherwise we won't observe this spurious relationship between the exposure and outcome.

Reply:
by surrogate i think it means the risk is measured by a proximal factor. like if tar in lungs is the real cause of lung cancer in smokers, and you can't measure tar in lungs, you have to approximate it by measuring number of cigarettes smoked.

Reply:
Thanks for all these clarifying comments, Nicole!